policy/aggregation_type_options.go

```diff
@@ -159,6 +159,15 @@ type AggregationTypesOptions interface {
 	// Suffix returns the suffix for the aggregation type for gauges.
 	SuffixForGauge(value AggregationType) []byte
 
+	// AggregationTypeWithSuffixForCounter returns the aggregation type with the given suffix for counters.
+	AggregationTypeWithSuffixForCounter(value []byte) AggregationType
```

Comment:
Perhaps just call it `AggregationTypeForCounter(value []byte)`? Same for the rest of these.